visual-inspection/inspection-backend/inspection/model/model.py

```diff
@@ -1,21 +1,70 @@
 import tensorflow as tf
 
 # TODO: Replace with custom model variant
-model = tf.keras.applications.MobileNetV2(input_shape=None,
-                                          include_top=True,
-                                          weights="imagenet")
-
+#model = tf.keras.applications.MobileNetV2(input_shape=None,
+#                                          include_top=True,
+#                                          weights="imagenet")
+model = tf.keras.applications.Xception( include_top=True,
```

Comment:
```suggestion
model = tf.keras.applications.Xception(include_top=True,
```

(Minor issue: Please adhere to PEP8 style guide, e.g., run an auto-formatter or check with flake8)